src/App.js

```diff
@@ -30,6 +30,11 @@ const style = css`
     background-color: #ffffff;
     box-shadow: 0 4px 7px 0 rgba(0, 0, 0, 0.15);
   }
+  @media only screen and (min-device-width: 320px) and (max-device-width: 480px) {
+    div.App {
+      min-width: 100%;
```

Comment:
General idea. Why if instead of specifying `margin` for every `div` in the mobile screen, we instead add a `padding-left:16px` and `padding-right:16px` right here in `div.App` ? And set all other margins in the rest of div's to 0?